Makefile

```diff
@@ -3,7 +3,7 @@
 BUILD_DIR=build
 SPARQL=sparql
 ROBOT_ENV=ROBOT_JAVA_ARGS=-Xmx12G
-ROBOT=$(ROBOT_ENV) robot
+ROBOT=$(ROBOT_ENV) ./robot
```

Comment:
I don't think you should specify the path to robot like this—it's the responsibility of the user/environment to have it in the PATH.

Reply:
`robot` was in path but I was still getting error `/bin/sh: robot: command not found`